scripts/~Settings.nut

```diff
@@ -0,0 +1,7 @@
+local gt = this.getroottable();
+
+// This value determines whether more than one gold mine can be built per stronghold
+gt.Stronghold.AllowMoreThanOneGoldMine <- true; // Boolean value, true or false
```

Comment:
Value should be false by default

Reply:
yep. was done in previous commit, dunno why i left it to `true`. my bad, brainfart.